homeworks/07-mars-viewer/src/features/mars-viewer/photos/Photos.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { createSelector } from '@reduxjs/toolkit';
+
+import { useAppSelector, useAppDispatch } from '../../../app/hooks';
+import { PhotoGrid } from '../photo-grid/PhotoGrid';
+import {
+    selectSelectedSol,
+    selectSols,
+    selectPhotos,
+    changeSelectedSol,
+    fetchSol
+} from '../marsSlice';
+import style from './Photos.module.css';
+
+export const Photos: React.FC = () => {
+    const dispatch = useAppDispatch();
+    const selectedSol = useAppSelector(selectSelectedSol);
+    const photosSelector = createSelector(
+        selectSelectedSol,
+        selectSols,
+        selectPhotos,
+        (selectedSol, sols, photos) => {
+            const photoIds = sols[selectedSol];
+            if (photoIds === undefined) {
+                return null;
```

Comment:
from my perspective it's better return empty array instead of null in js, typescript will handle this case, but in projects with js or partially converted to ts it may lead to runtime errors